test_runner/src/main/kotlin/ftl/adapter/GcStorageExists.kt

```diff
@@ -0,0 +1,10 @@
+package ftl.adapter
+
+import ftl.adapter.google.GcStorage
+import ftl.api.RemoteStorage
+
+object GcStorageExists :
+    RemoteStorage.Exist,
+    (RemoteStorage.Dir) -> Boolean by {
+        GcStorage.exist(it.bucket, it.path)
```

Comment:
should not we return it on client layer ?

Reply:
@piotradamczyk5 Can i assume that this is the toAPI that i see in your PRS correct?

Reply:
@piotradamczyk5 I agree, it should be moved to the `client` package sooner or later.

Reply:
@piotradamczyk5 @jan-gogo Do we have a good example how this was done? I am confused what you mean here.

Reply:
Done